config/_shared.json

```diff
@@ -129,5 +129,11 @@
 	"wpcom_signup_key": false,
 	"statsd_analytics_response_time_max_logs_per_second": 50,
 	"google_maps_and_places_api_key": "AIzaSyAJN-ceOpTLP1CSQdyoRLS4VT9Zlsibkeg",
-	"push_notification_vapid_key": "BDQlTOrVTrxRM0i-gCsrxjxOyul281c5WVdfstXaqZqTSpjSI9M-E99CvwwJRWDkZ0goa8VSqUCgwiexxcDuXCs"
+	"push_notification_vapid_key": "BDQlTOrVTrxRM0i-gCsrxjxOyul281c5WVdfstXaqZqTSpjSI9M-E99CvwwJRWDkZ0goa8VSqUCgwiexxcDuXCs",
+	"disable_sections": false,
```

Comment:
This flag feels confusing to me. Which sections are we not disabling here? Why do I need to introduce just another setting to disable the sections in the main entry point?

I'd advise just going with the `sections` setting, and disabling all sections for the Jetpack Cloud environments instead.

Reply:
The reason for this is that I didn't want to have to have a huge list of all sections. 
I think renaming this to `enable_sections` or `enable_sections_by_default` and set it to true would make things a lot clearer.